pyam/core.py

```diff
@@ -757,26 +758,30 @@ def rename(self, mapping=None, inplace=False, append=False,
         if not inplace:
             return ret
 
-    def convert_unit(self, conversion_mapping, inplace=False):
-        """Converts units based on provided unit conversion factors
+    def convert_unit(self, current, to=None, factor=None, inplace=False):
+        """Converts a unit using given factor or pint.UnitRegistry
+
+        You can access the UnitRegistry using :meth:`pyam.units.registry()`.
 
         Parameters
         ----------
-        conversion_mapping: dict
-            for each unit for which a conversion should be carried out,
-            provide current unit and target unit and conversion factor
-            {<current unit>: [<target unit>, <conversion factor>]}
+        current: str (or mapping, deprecated)
+            name of current unit (to be converted from)
+        to: str
+            name of new unit (to be converted to)
+        factor: value, optional
+            conversion factor to used, defaults to pint.UnitRegistry
```

Comment:
"to use" is grammatically correct, possibly 'to multiply by' is clearer. This section is a little confusing because of the use of 'to' as a key word!

Reply:
changed